src/test/java/api/support/fixtures/NoticeMatchers.java

```diff
@@ -32,14 +38,22 @@ private NoticeMatchers() {
     return tokenMatchers;
   }
 
-  public static Map<String, Matcher<String>> getItemContextMatchers(IndividualResource itemResource) {
+  public static Map<String, Matcher<String>> getItemContextMatchers(InventoryItemResource itemResource,
```

Comment:
As this grows, I increasingly find it harder to follow what is being checked, especially now there is also branching logic.

I wonder if this could be split up and the checks made more explicit. Maybe similar to the composition used by the `ValidationErrorMatchers` or the `RequestItemMatcher`.

What do you think?

This may be out of scope for this work. I wanted to raise the suggestion anyway.